src/Compilers/CSharp/Portable/Compilation/SyntaxAndDeclarationManager.cs

```diff
@@ -548,6 +580,9 @@ public SyntaxAndDeclarationManager ReplaceSyntaxTree(SyntaxTree oldTree, SyntaxT
 
                 ordinalMap = ordinalMap.Remove(oldTree);
                 ordinalMap = ordinalMap.SetItem(newTree, oldOrdinal);
+
+                lastComputedMemberNamesMap.Remove(oldTree);
```

Comment:
>lastComputedMemberNamesMap.Remove(oldTree); [](http://example.com/codeflow?start=16&length=43)

Is this necessary? Is ```oldTree``` going to be in ```removeSet``` as well?

Reply:
Likely redundant.